### PR TITLE
Automate OCP-13016 (node hpa upgrade test)

### PR DESCRIPTION
**Pull request by weinliu**
Running log: http://pastebin.test.redhat.com/865458

@jhou1 @sunilcio @lyman9966 could you help with the review? Thanks.
cc @geliu2016

### CODE REVIEW

**Comment by weinliu** [comment]
@jhou1 @lyman9966  @sunilcio I've got cases of OCP-13017,OCP-13019 merged (checking daemonset and configmap),  running log: http://pastebin.test.redhat.com/866296
Could you guys take another look?

File: features/upgrade/node/hpa/upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Feature: basic verification for upgrade testing
+  # @author weinliu@redhat.com
+  @upgrade-prepare
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure hpa works well after upgrade - prepare
```
Comment (jhou1) [inline features/upgrade/node/hpa/upgrade.feature:6]: Can you please update the scenario name for the test covers multiple resources? For example: `Upgrade - Make sure multiple resources works well after upgrade - prepare`

**Comment by jhou1** [comment]
Yes @weinliu no problem. The scenario needs renaming, other than that LGTM.
Please also combine the 3 tests in polarion(update one and deactivate the rest), thanks.

File: features/upgrade/node/hpa/upgrade.feature (hunk 1 above)
Comment (weinliu) [reply features/upgrade/node/hpa/upgrade.feature:6]: Got you, updated

**Comment by weinliu** [comment]
> Yes @weinliu no problem. The scenario needs renaming, other than that LGTM.
> Please also combine the 3 tests in polarion(update one and deactivate the rest), thanks.

Now the three cases have been merged into one

File: features/upgrade/node/hpa/upgrade.feature (hunk 1 above)
Comment (jhou1) [reply features/upgrade/node/hpa/upgrade.feature:6]: The prepare step also needs updating

File: features/upgrade/node/hpa/upgrade.feature [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    Then the step should succeed	
+    And the pod named "dapi-test-pod-1" status becomes :succeeded
+ 
+
+  # @author weinliu@redhat.com
+  # @case_id OCP-13016,OCP-13017,OCP-13019
+  @upgrade-check
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure multiple resources work well after upgrade - prepare
```
Comment (jhou1) [inline features/upgrade/node/hpa/upgrade.feature:46]: The `- prepare` needs to be removed from the scenario as this is the upgrade-check step

Comment (jhou1) [reply features/upgrade/node/hpa/upgrade.feature:46]: You should follow a convention like this for prepare and check
```
Scenario: etcd-operator and cluster works well after upgrade - prepare
.....

Scenario: etcd-operator and cluster works well after upgrade
```

Comment (weinliu) [reply features/upgrade/node/hpa/upgrade.feature]: > You should follow a convention like this for prepare and check, for example
> 
> ```
> Scenario: etcd-operator and cluster works well after upgrade - prepare
> .....
> 
> Scenario: etcd-operator and cluster works well after upgrade
> ```

Thanks, they are now in a converion pair

File: features/upgrade/node/hpa/upgrade.feature [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      | n | <%= project.name %>                                       |
+    Then the step should succeed	
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/configmap/pod-configmap-volume1.yaml |
+    Then the step should succeed	
+    And the pod named "dapi-test-pod-1" status becomes :succeeded
+ 
+
+  # @author weinliu@redhat.com
+  # @case_id OCP-13016,OCP-13017,OCP-13019
```
Comment (liangxia) [inline features/upgrade/node/hpa/upgrade.feature:42]: We should leave only one test case id there

Comment (jhou1) [reply features/upgrade/node/hpa/upgrade.feature:42]: Nice catch!

Comment (weinliu) [reply features/upgrade/node/hpa/upgrade.feature]: > We should keep only one test case id there

Updated, let me know if it's good to go. Thanks!

**Pull request merged**